diesel/src/expression/count.rs

```diff
@@ -1,9 +1,11 @@
-use super::functions::sql_function;
+use std::marker::PhantomData;
+
+use super::{functions::sql_function, is_aggregate, AsExpression};
```

Comment:
This should follow the general use statement style in diesel

```suggestion
use super::functions::sql_function;
use super::{is_aggregate, AsExpression};
```